src/pykeen/datasets/wd50k.py

```diff
@@ -55,10 +72,154 @@ def __init__(self, **kwargs):
         )
 
 
+@parse_docdata
+class WD50K(HyperRelationalUnpackedRemoteDataset):
+    """The hyper-relational version of WD50K.
+
+    ---
+    name: WD50K (hyper-relational)
+    citation:
+        author: Galkin
+        year: 2020
+        link: https://www.aclweb.org/anthology/2020.emnlp-main.596/
+        arxiv: 2009.10847
+        github: migalkin/StarE
+    statistics:
+        entities: 47,156 (5,460 qualifier-only)
```

Comment:
@cthoyt do we rely somewhere on parsing these statistics to integers? If yes, we should store the second number under a different key